packages/frontend/core/src/components/blocksuite/block-suite-header/favorite/index.tsx

```diff
@@ -30,11 +31,16 @@ export const useFavorite = (pageId: string) => {
 export const FavoriteButton = ({ pageId }: FavoriteButtonProps) => {
   const { favorite, toggleFavorite } = useFavorite(pageId);
 
+  const handleFavorite = useCallback(() => {
+    track.$.header.actions.toggleFavorite();
```

Comment:
simple event can tracked by data-event-props

Reply:
I tried this, but it didn't work.
`data-event-props="$.header.actions.toggleFavorite"`

Reply:
It looks like a problem with IconButton